code/datums/components/connect_loc_behalf.dm

```diff
@@ -19,17 +19,6 @@
 	src.connections = connections
 	src.tracked = tracked
 
-/datum/component/connect_loc_behalf/CheckDupeComponent(datum/component/component, atom/movable/tracked, list/connections)
```

Comment:
connect_loc_behalf has a COMPONENT_DUPE_UNIQUE dupe_mode, while the CheckDupeComponent is only called if it's COMPONENT_DUPE_SELECTIVE.